<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="restartAlert">
     <property name="styleSheet">
      <string notr="true">background: #c0392b;
color: #fcfcfc;
font-weight: bold;
padding: 5px;
border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>Server must be restarted for changes to take effect</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Server</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User Interface:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="trayIcon">
        <property name="text">
         <string>Show tray status icon</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Session storage:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="storageMemory">
        <property name="text">
         <string>Memor&amp;y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="storageFile">
        <property name="text">
         <string>Fi&amp;le</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox">
     <property name="title">
      <string>Network</string>
     </property>
     <layout class="QFormLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="port">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="value">
         <number>27750</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Services:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Local hostname:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="localAddress"/>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="zeroconf">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>ZeroConf announcement</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="upnp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Automatic UPnP port forwarding</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Security</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="toolTip">
         <string>URL of the external authentication server</string>
        </property>
        <property name="text">
         <string>Ext-auth server:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="extAuthUrl"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>TLS:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Certificate:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="certFile">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pickCert">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Key:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="keyFile">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pickKey">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="enableTls">
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableTls</sender>
   <signal>toggled(bool)</signal>
   <receiver>certFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableTls</sender>
   <signal>toggled(bool)</signal>
   <receiver>pickCert</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableTls</sender>
   <signal>toggled(bool)</signal>
   <receiver>keyFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableTls</sender>
   <signal>toggled(bool)</signal>
   <receiver>pickKey</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
